composer.json

```diff
@@ -30,7 +30,7 @@
     "phpunit/dbunit": "1.3.*",
     "phpunit/phpunit-story": "1.0.*",
     "phpunit/php-invoker": "1.1.*",
-    "ocramius/instantiator": "1.1.4"
+    "doctrine/instantiator": "1.1.4"
```

Comment:
I am getting a "doctrine/instantiator could not be found in any version" message from composer. I believe the Solr 5.x problems are caused by an old version of Solarium. That is why I tried running "composer update".